frb_example/pure_dart_multi/rust/src/block_specific_module/for_api_block_2.rs

```diff
@@ -0,0 +1,19 @@
+/// This is a struct only used in API block 2 for test, but not defined in block file
+pub struct StructOnlyForBlock2 {
+    pub id: i16,      // in-built type only used in API block 2 for test
```

Comment:
nit: I am worried, how can we ensure i16 does not appear in anywhere else accidentially in the future?

(the answer can be just a simple "no easy approach"!)

Reply:
>  I am worried, how can we ensure i16 does not appear in anywhere else accidentially in the future?

That is why I added this comment(and others similar to this)--- Technically,  there is no way to restrict anyone, including me to use `StructOnlyForBlock2` in other situations. But better than none, when it is to be used in other situations, this comment would be useful to warn that this struct is not aimed to be used in situations other than that said in the comments.   

For the specific content, the comment is meant to say that the struct is designed specifically to test in API block 2 only for testing purposes.